src/get-files-stream.js

```diff
@@ -82,7 +82,8 @@ function loadPaths (opts, file) {
         // files inside symlinks and others
         return
       })
-      .filter((file) => !!file) // filter out null files
+      // filter out null files
+      .filter((f) => Boolean(f))
```

Comment:
That's how "type casting" in js is done, exists for all the types `Number`, `Boolean`, `String`.

Yes we can do.